core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -1388,20 +1388,23 @@ class KafkaApisTest extends Logging {
       topics.foreach(topic => {
         val metadataRequestData = new MetadataRequestData().setTopics(Collections.singletonList(topic))
         val request = buildRequest(new MetadataRequest(metadataRequestData, version.toShort))
-        kafkaApis = createKafkaApis()
```

Comment:
This `kafkaApis` will be created multiple times in this test, so we have to close them 1 by 1 inside the test.